django-verdant/rca/templates/rca/includes/modules/share-button.html

```diff
@@ -6,7 +6,8 @@
 		<h2>Share</h2>
 		<ul>
 			<li><a href="https://www.facebook.com/sharer/sharer.php?u={{ self.url }}">On Facebook</a></li>
-			<li><a href="http://twitter.com/share?text={{ self.title }}&url={{ self.url }}&via={{ global_default_twitter_handle }}">On Twitter</a></li>
+			<li><a href="https://twitter.com/share?text={{ self.title }}&url={{ self.url }}&via={{ global_default_twitter_handle }}">On Twitter</a></li>
+			<li><a href="https://www.linkedin.com/shareArticle?mini=true&url={{ self.url }}">On LinkedIn</a></li>
```

Comment:
I am actually wondering if we should not use `|urlencode` on URLs. Even if the other ones don't have it. Can you please check this? Thanks.

Reply:
I've added the urlencode and tested the links now.